### PR TITLE
[Bug Form] Bug Form Update

### PR DESCRIPTION
**Pull request by jaredmullane**
Signed-off-by: Jared Mullane <jmullane@fb.com>

## Summary

Adding a line to the "bug report" workflow for contributors to add a thumbs-up emoji to an open issue if it matches their current one rather than submitting a new issue. This allows us to filter by thumbs-up emojis to understand how wide spread an issue is.

## Test Plan

Minor markdown change

### CODE REVIEW

**Review by hcgatewood — APPROVED** [review]

**Review by hcgatewood — APPROVED** [review]
Add newline between the change and the `---`

**Comment by ekowtaylor** [comment]
@jaredmullane Once this PR has been approved and passing all CI checks, use the ready2merge label to indicate the maintainers can merge your PR.

**Comment by jaredmullane** [comment]
Updated

**Comment by themarwhal** [comment]
Hey @jaredmullane ! Can you rebase and amend the commit like this?
`git commit --amend --signoff --allow-empty` the DCO check is failing.

**Comment by jaredmullane** [comment]
Thanks @themarwhal! I ended up creating a new PR here: https://github.com/magma/magma/pull/7312 but should be good to go on the DCO checks.

**Pull request closed without merging**
